### ISSUE TITLE
The identity GNMI_STREAMING_OBJECT_TYPE does not use tapi-common:OBJECT_TYPE as its base

### ISSUE THREAD

**Issue by roshan-joyce-fujitsu** [Open-Network-Models-and-Interfaces-ONMI/TAPI#589]
The identity GNMI_STREAMING_OBJECT_TYPE is defined as below without any base:
```
    identity GNMI_STREAMING_OBJECT_TYPE {
        description "none";
    }
```

And GNMI_STREAMING_OBJECT_TYPE_MEASUREMENT_DETAILS is defined as:
```
    identity GNMI_STREAMING_OBJECT_TYPE_MEASUREMENT_DETAILS {
        base GNMI_STREAMING_OBJECT_TYPE;
        description "none";
    }
```

I am suspecting this will create a Yang modeling issue when representing a `supported-stream-type` object with `stream-type-content` as "GNMI_STREAMING_OBJECT_TYPE_MEASUREMENT_DETAILS" because the type of `stream-type-content` is tapi-common:object-type.
```
       +--ro supported-stream-type* [uuid]
          +-- ...
          +--ro stream-type-content*           tapi-common:object-type
```

Please note that all other object-type identities use tapi-common:OBJECT_TYPE as their base. Please see 2 examples below:
```
    identity CONNECTIVITY_OBJECT_TYPE {
    	base tapi-common:OBJECT_TYPE;
        description "none";
    }
```
```
    identity STREAMING_OBJECT_TYPE {
        base tapi-common:OBJECT_TYPE;
        description "none";
    }
```
CC: @amazzini @nigel-r-davis

**Comment by amazzini** [comment]
Yes looks like an oversight to be fixed. To be discussed how, e.g. delivering a 2.5.2 patch version or waiting for 2.6.0

**Comment by amazzini** [comment]
Fixed in .yang candidate for 2.6.0

**Comment by roshan-joyce-fujitsu** [comment]
This is now fixed in 2.6.0.
